draft-ietf-quic-transport.md

```diff
@@ -1714,12 +1714,6 @@ meets the following criteria:
   acknowledgment for a packet containing a PATH_CHALLENGE frame is not adequate
   validation, since the acknowledgment can be spoofed by a malicious peer.
 
-- It was sent from the same remote address to which the corresponding
-  PATH_CHALLENGE was sent. If a PATH_RESPONSE frame is received from a different
-  remote address than the one to which the PATH_CHALLENGE was sent, path
-  validation is considered to have failed, even if the data matches that sent in
-  the PATH_CHALLENGE.
-
 - It was received on the same local address from which the corresponding
   PATH_CHALLENGE was sent.
```

Comment:
I think that we can remove this requirement also.  For the recipient, there is no requirement to have the packet that carries a PATH_RESPONSE travel on any particular path.

What you need to have here is a much clearer split in the requirements: the responder MUST send a PATH_RESPONSE on the same path that it got the PATH_CHALLENGE on, but the initiator MUST accept a PATH_CHALLENGE that arrives on any path.  I think that you need a paragraph explaining that split and why it exists.

Reply:
Makes sense, thanks! Will add.